app/gui/src/project-view/components/SelectionDropdown.vue

```diff
@@ -8,13 +8,15 @@ import SvgIcon from '@/components/SvgIcon.vue'
 import type { SelectionMenuOption } from '@/components/visualizations/toolbar'
 import { ref, toValue } from 'vue'
 
-type Key = number | string | symbol
+type Key = string
+
 const selected = defineModel<Key>({ required: true })
-const _props = defineProps<{
+const { options, title, labelButton, alwaysShowArrow, entriesTestId } = defineProps<{
```

Comment:
If they are used in template only, we may also not declare return vars at all:
```suggestion
defineProps<{
```